src/Logging/test/ConsoleLoggerTest.cs

```diff
@@ -166,136 +157,11 @@ public void LogsWhenNullFilterGiven()
             Assert.Equal(expectedHeader + expectedMessage, sink.Writes[1].Message);
         }
 
-        [Fact]
-        public void CriticalFilter_LogsWhenAppropriate()
```

Comment:
Any reason these are gone?

Reply:
No more filtering directly in console logger.